pkg/operator/ceph/cluster/mon/service.go

```diff
@@ -67,6 +69,30 @@ func (c *Cluster) createService(mon *monConfig) (string, error) {
 		return "", nil
 	}
 
-	logger.Infof("mon %q ip is %s", mon.DaemonName, s.Spec.ClusterIP)
+	if c.spec.Network.MultiClusterService.Enabled {
```

Comment:
looks this this is only eligible for normal cluster,
Won't we have a use case for external cluster?

Reply:
I don't believe we need it for the external case because we won't create Mon or OSD services for external clusters.